frontend/game/hand.js

```diff
@@ -28,7 +13,7 @@ define(function(require) {
                 var newCard = helper.baseCard.cloneNode(true);
                 newCard.innerHTML = cardInHand["power"];
                 newCard.setAttribute("index", index);
-                newCard.addEventListener('click', function(){clickCard(newCard)});
+                newCard.addEventListener('click', function(){play.clickCard(newCard)});
```

Comment:
I have the feeling this could be prettier, but don't know how.

Reply:
```javascript
newCard.addEventListener('click', () => play.clickCard(newCard) );
```
maybe?

Reply:
I saw this alternative on SE.
But it seems equally ugly + harder to understand.